components/calendar/src/astronomy.rs

```diff
@@ -395,24 +394,30 @@ impl Astronomical {
         )
     }
 
-    /// Calculates the right ascension at a given [`Moment`] of UTC time of an object at latitude `beta` and longitude `lambda`; all angles are in degrees.
+    /// Calculates the right ascension at a given [`Moment`] of UTC time of an object at ecliptic latitude `beta` and ecliptic longitude `lambda`; all angles are in degrees.
     /// the right ascension is the angular distance east or west of an object in the sky with respect to the plane
     /// of the vernal equinox, which is the celestial coordinate point at which the ecliptic intersects the celestial
     /// equator marking spring in the northern hemisphere; analogous to longitude.
     ///
+    /// This will return None for an object at the ecliptic pole. Note that this is not when latitude = 90°,
+    /// since that point is 90° from the celestial equator, not from the plane of the ecliptic.
```

Comment:
```suggestion
    /// This will return None for an object at the celestial pole. Note that this is not when β = 90°,
    /// since the celestial pole is 90° from the celestial equator, not from the plane of the ecliptic.
```